Dockerfile

```diff
@@ -36,54 +33,43 @@ RUN if [ "${BUILD_DATE}" = "undefined" ]; then \
   npm run build -- --base=$UI_PUBLIC_URL
 
 ### Build API ###
-FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.22-bookworm AS api
+FROM --platform=${BUILDPLATFORM:-linux/amd64} golang:1.22-alpine AS api
 ARG TARGETPLATFORM
 
-# See for details: https://github.com/hadolint/hadolint/wiki/DL4006
-SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
-
 WORKDIR /app/api
 
 ENV GOPATH="/go"
 ENV PATH="${GOPATH}/bin:${PATH}"
 ENV CGO_ENABLED=1
 
 # Download dependencies
-COPY scripts/apt/debian-testing.sources /etc/apt/sources.list.d/
-COPY scripts/set_compiler_env.sh /app/scripts/
 COPY scripts/install_build_dependencies.sh /app/scripts/
 COPY scripts/install_runtime_dependencies.sh /app/scripts/
 RUN chmod +x /app/scripts/*.sh \
   && /app/scripts/install_build_dependencies.sh \
+  && mv $(go env GOPATH)/bin/reflex /usr/bin/ \
   && /app/scripts/install_runtime_dependencies.sh
 
 COPY api/go.mod api/go.sum /app/api/
-RUN source /app/scripts/set_compiler_env.sh \
-  && go env \
+RUN go env \
```

Comment:
Why don't we need to source the set env script?

Reply:
The `set_env` script is used for cross-compiling. As we are building `arm` images in QEMU env, it does not require any cross-compiling. It's better to be removed to keep the workflow simple. Also, it's more stable to rely on the predefined env variables than to set them manually.